BUILD.bazel

```diff
@@ -20,12 +30,14 @@ SDL_LINKOPTS = select({
 		"-DEFAULTLIB:hid",
 		"-DEFAULTLIB:setupapi",
 		"-DEFAULTLIB:opengl32",
+		"-DEFAULTLIB:kernel32",
 	],
 	"//conditions:default": [],
 })
 
 cc_library(
 	name = "SDL2_hdrs",
+	include_prefix = "SDL2",
```

Comment:
Isn't the inclusion of SDL2 typically done with

```cpp
#include "SDL.h"
// or
#include <SDL.h>
```

Not

```cpp
#include "SDL2/SDL.h"
// or
#include <SDL2/SDL.h>
```

```suggestion
```

Reply:
Yep, I've been using it wrong for a long time. Well spotted again!